src/stravalib/client.py

```diff
@@ -528,7 +528,7 @@ def get_club(self, club_id: int) -> model.Club:
 
     def get_club_members(
         self, club_id: int, limit: int | None = None
-    ) -> BatchedResultsIterator[model.Athlete]:
+    ) -> BatchedResultsIterator[strava_model.ClubAthlete]:
```

Comment:
This is a new type introduced by Strava a while ago. We never updated this client method to reflect that update.